server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -151,6 +172,35 @@ private static Map<String, UploadedMetadataAttribute> customMetadata(Object[] fi
         return customs.stream().collect(Collectors.toMap(UploadedMetadataAttribute::getAttributeName, Function.identity()));
     }
 
+    private static UploadedMetadataAttribute discoveryNodesMetadata(Object[] fields) {
```

Comment:
Can we merge this from main, routing table related fields have already gone in and we will need to change field numbering to merge this.